force-app/main/default/lwc/daysOnMarket/daysOnMarket.css

```diff
@@ -6,13 +6,9 @@
 }
 
 .badge.alert {
-    background-color: rgb(
-        194,
-        57,
-        52
-    ); /* Using RGB as var(--lwc-colorBackgroundDestructive); is internal */
+    background-color: rgb(194, 57, 52);
     color: var(--lwc-colorTextInverse, #ffffff);
-    border: solid 1px rgb(194, 57, 52); /* Using RGB as var(--lwc-colorBorderDestructive); is internal */
+    border: solid var(--lwc-borderWidthThin, 1px) rgb(194, 57, 52);
```

Comment:
@albarivas can you double check whether we're allowed to use `--lwc` CSS variables?
My understanding (which aligns with the previous code's comments) is that these are internal and may change or get removed without notice unlike the documented Styling Hooks.
I'm also worried that these CSS variables are not documented on the [SLDS website](https://www.lightningdesignsystem.com).